2.7/Dockerfile.rhel7

```diff
@@ -11,16 +11,28 @@ ENV PYTHON_VERSION=2.7 \
     LANG=en_US.UTF-8 \
     PIP_NO_CACHE_DIR=off
 
+ENV SUMMARY="Platform for building and running Python PYTHON_VERSION applications" \
+    DESCRIPTION="Python $PYTHON_VERSION available as docker container, is a base platform for \
+building and running various Python $PYTHON_VERSION applications and frameworks. \
+Python is an easy to learn, powerful programming language. It has efficient high-level \
+data structures and a simple but effective approach to object-oriented programming. \
+Python’s elegant syntax and dynamic typing, together with its interpreted nature, \
+make it an ideal language for scripting and rapid application development in many areas \
+on most platforms."
+
+LABEL summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$SUMMARY" \
+      io.k8s.display-name="Python 2.7" \
+      io.openshift.expose-services="8080:http" \
+      io.openshift.tags="builder,python,python27,rh-python27"
+
 # Labels consumed by Red Hat build service.
 LABEL com.redhat.component="python27-docker" \
       name="rhscl/python-27-rhel7" \
       version="2.7" \
       release="1" \
```

Comment:
>What constitutes a reason to do a release bump in containers?
afaik we do not do a release bump in upstream Dockerfiles.
The existing release values > 1 in other containers are probably changes that got upstream from rhel images, where we need to do proper versioning (major release bump for devel->beta, beta->GA and minor release bump for rebuilds).